platform/podjs/README.md

```diff
@@ -29,12 +29,106 @@ soon.
 
 Simply run `build.js` in the repository root.
 
-## Usage
+## How to setup a feature to build with podjs
 
-You will need to use this module for creating a polyPod-compatible
-feature. Follow these instructions:
+To create a polyPod compatible feature, you will need to use this module.
+Follow these instructions to do so:
 
-1. Add _podjs_ as a development dependency to your feature.
-2. Copy or link `node_modules/@polypoly-eu/podjs/dist/pod.js`.
-3. Include `pod.js` from your feature's main document - before the
-   feature's own code.
+1.  In order to point to `podjs` from the polypoly-eu repository,
+    add as `development dependencies` on your `package.json` the following:
+
+      <code>
+
+         "devDependencies": {
+            "@polypoly-eu/podjs": "file:../../platform/podjs",
+         },
+
+      </code>
+
+2.  We use `rollup` module bundler to build our features all together via the script
+
+    <code> $ rollup -c rollup.config.json </code>
+
+    which runs `Rollup` and takes the `./out-tsc` directory as input and
+    puts the result in a `dist` folder. Those configurations need to be defined
+    in the `rollup.config.js` file.
+
+    We basically rely on our in-house `rollup-plugin-copy-watch` plugin
+    on top of the `Rollup` one which serves to copy files and folders with glob support
+    and also offers an additional watch to other sources than just Rollup's bundle content.
+    This is also needed to be installed as a dev dependency:
+
+      <code>
```

Comment:
Is this what's causing the prettier to complain? Have you tried using code fencing, like this?

```
```typescript
```
```